arrow-fx-android/gradle.properties

```diff
@@ -0,0 +1,5 @@
+# Maven publishing configuration
+POM_NAME=Arrow-Fx-Android
+POM_ARTIFACT_ID=arrow-fx-android
+POM_PACKAGING=aar
```

Comment:
```suggestion
POM_PACKAGING=jar
```

Reply:
@rachelcarmena I personally choose aar since android libs work this way additionally, so it's like a requirement to the project implementing to be an android lib/app, does it make sense? Otherwise we can revert to jar since it's compatible anyway 🤷

Reply:
Oh, ok! I thought it was a typo :joy: